python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S4507.html

```diff
@@ -32,12 +32,23 @@ <h2>Sensitive Code Example</h2>
 </pre>
 <p>Django's "global_settings.py" configuration file</p>
 <pre>
-# NOTE: The following code raises issues only if the file is named "global_settings.py". This is the default
+# NOTE: The following code raises issues only if the file is named "settings.py". This is the default
 # name of Django configuration file
 
 DEBUG = True  # Sensitive
 DEBUG_PROPAGATE_EXCEPTIONS = True  # Sensitive
 </pre>
+<p>Flask</p>
+<pre>
+from flask import Flask
+
+app = Flask(__name__)
+
+app.testing = True  # Questionable
+app.debug = True  # Questionable
+
+app.run(debug=True) # Questionable
```

Comment:
Do we really raise issues on those Flask cases?

Reply:
Indeed, we don't support them. I will remove this update. Thanks!